cargo-shuttle/src/init.rs

```diff
@@ -390,8 +395,10 @@ fn set_key_value_dependency_version(
     manifest_path: &Path,
     url: &Url,
     get_dependency_version_fn: GetDependencyVersionFn,
+    flag_allow_prerelease: bool,
 ) {
-    let dependency_version = get_dependency_version_fn(crate_name, manifest_path, url);
+    let dependency_version =
+        get_dependency_version_fn(crate_name, flag_allow_prerelease, manifest_path, url);
```

Comment:
Made the `flag_allow_prerelease` bool as a function argument for `set_value_dependency_version` --which sets the version as `crate_name = "version"`-- to be able to pass it within the implementation of each framework. However, now the `set_inline_table_dependency_version` --which sets the version as `crate_name = { version = "version" }`-- does not receive the same arguments, and becomes asymmetrical. 

Should I also update the latter?

Reply:
Nvm, I just answered my own question :facepalm: